packages/core/src/index.ts

```diff
@@ -151,17 +157,18 @@ export function apply(ctx: Context, config: Config) {
         count: options.count,
         labels: options.label?.split(',')?.map((x) => x.trim())?.filter(Boolean) ?? [],
       })
+      const source = images.source
 
       if (!images || !images.length) return session?.text('.no-result')
 
-      images = images.filter((image) => config.nsfw || !image.nsfw)
+      images = images.filter((image) => config.nsfw || !image.nsfw) as ImageArray
```

Comment:
Does the `as`-cast still required even the `get` function return a proper type?

Reply:
yes